packages/mdc-chips/chip/adapter.ts

```diff
@@ -107,7 +114,42 @@ export interface MDCChipAdapter {
   getCheckmarkBoundingClientRect(): ClientRect | null;
 
   /**
-   * Sets the value of the attribute on the root element.
+   * Sets the value of the attribute on the primary action content.
+   */
+  setTextAttr(attr: string, value: string): void;
+
+  /**
+   * @return true if the primary action content has focus.
+   */
+  textHasFocus(): boolean;
+
+  /**
+   * Gives focus to the primary action content.
+   */
+  focusText(): void;
+
+  /**
+   * @return true if the chip has a trailing action.
+   */
+  hasTrailingIcon(): boolean;
+
+  /**
+   * Returns true if the primary action content has focus.
```

Comment:
This says returns true - actually I think this is an incorrect description. Please also add `@param`